diff --git a/src/Util/Cache.php b/src/Util/Cache.php
--- src/Util/Cache.php
+++ src/Util/Cache.php
@@ -263,9 +263,11 @@
         }

         if (file_exists(self::$path) === true) {
             self::$cache = json_decode(file_get_contents(self::$path), true);
-
+            if ( ! \is_array( self::$cache )) {
+				self::$cache['config'] = [];
+			}
             // Verify the contents of the cache file.
             if (self::$cache['config'] !== $configData) {
                 self::$cache = [];
                 if (PHP_CODESNIFFER_VERBOSITY > 1) {
